mainnet/assets.json

```diff
@@ -818,100 +818,28 @@
       },
       "extra": {}
     },
-    {
```

Comment:
these two tokens should not be removed in your changes, you should just add your token at the bottom